doc/src/manual/performance-tips.md

```diff
@@ -1394,6 +1394,116 @@ Prominent examples include [MKL.jl](https://github.com/JuliaLinearAlgebra/MKL.jl
 These are external packages, so we will not discuss them in detail here.
 Please refer to their respective documentations (especially because they have different behaviors than OpenBLAS with respect to multithreading).
 
+## Execution latency, loading and precompiling time
+
+### Reducing time to first plot etc.
+
+The first time a julia function is called it will be compiled. The [`@time`](@ref) macro family illustrates this.
```

Comment:
"it" and all its callees that can be statically determined, right? May be nitpicking, but that's the reason why some small function can have a super long compile time.